pyproject.toml

```diff
@@ -147,7 +147,7 @@ synapse_review_recent_signups = "synapse._scripts.review_recent_signups:main"
 update_synapse_database = "synapse._scripts.update_synapse_database:main"
 
 [tool.poetry.dependencies]
-python = "^3.7.1"
+python = "^3.8.0"
```

Comment:
Which version of Python 3.8 do we want to specify here? For some reason we specified Python 3.7.1 before?

Reply:
It used to be 3.7.0, but then twisted made their min supported python version 3.7.1. The easiest way to keep Poetry happy was to copy them.

Reply:
Go with 3.8.0 if Poetry is happy with it.